traefik/test_traefikPrefixRouting.py

```diff
@@ -23,13 +24,16 @@ def test_routing():
     firstBackendPort   = traefikUtils.getPort("firstBackend")
     secondBackendPort  = traefikUtils.getPort("secondBackend")
 
-    traefik        = Popen(["traefik", "-c traefik.toml"],
+    dummyServerPath = abspath(join(dirname(__file__), 'dummyHttpServer.py'))
+    configFilePath  = abspath(join(dirname(__file__), 'traefik.toml'))
+
+    traefik        = Popen(["traefik", "-c", configFilePath],
                             stdout=PIPE)
-    defaultBackend = Popen(["python", "dummyHttpServer.py",str(defaultBackendPort)],
+    defaultBackend = Popen(["python", dummyServerPath, str(defaultBackendPort)],
```

Comment:
A useful trick here: use `sys.executable` as the first arg, then it is not sensitive to when the PATH might point to a different Python.

Reply:
You're totally right, I've just updated the pull request. Thanks a lot for the feedback!